spec/libs/custom_errors_spec.rb

```diff
@@ -15,7 +15,7 @@ def raise_500
   end
 
   before(:each) do
-    allow(Rails).to receive_message_chain(:env, :production?).and_return(true)
+    allow(Rails).to receive(:env) { 'production'.inquiry }
```

Comment:
Another way to change the environment. The one that it was being used it was failing.